### DISCUSSION TITLE
Add a disclaimer

### DISCUSSION THREAD

**Discussion by Spectre5** [robbievanleeuwen/section-properties#58] [General]
It is, of course, the users responsibility to perform their own due diligence and checks before blindly using this tool.  That said, I'd probably recommend making that explicitly stated in the README, something along the lines of "although efforts have been made to ensure correct results with this tool, it is provided as-is and it is the users responsibility to confirm the results.  The authors of this library cannot be held accountable for results obtained with this tool."  I'm sure there is much better wording you can find from another project, but the point is to make it clear that the tool is provided as-is with no guarantee on the accuracy.

**Comment by Czarified** [comment]
Is this not covered by the license we have? I guess it doesn't hurt to repeat it in the readme. I repeated it in my example page on #54 , since I'm talking about how accurate sectionproperties is.

**Reply by Spectre5** [reply]
Yes, I think you're right that the license does cover it.  I thought something explicit in the README may still be a good idea (maybe call it "guidance" or something besides a disclaimer).  But I'm okay if we don't too, maybe it is overkill.

**Reply by robbievanleeuwen** [reply]
Agreed that the license seems to cover this well, but I like the idea of adding a disclaimer, will push.

**Reply by robbievanleeuwen** [reply]
Updated with 4166d18. Locking this discussion.
